mochify/index.js

```diff
@@ -59,9 +59,7 @@ async function mochify(options = {}) {
     await shutdown(driver, server);
   }
 
-  if (exit_code) {
-    process.exitCode = exit_code;
-  }
+  return { exit_code };
```

Comment:
If my understanding of a catch-less `try / finally` is correct we do not need to set a default value here as we never get here in case `run` throws, right?